tests/test_app/TestApp/Model/Table/InvoicesTable.php

```diff
@@ -13,6 +13,7 @@ public function initialize(array $config)
             'contain' => [
                 'InvoiceItems.InvoiceItemProperties',
                 'InvoiceItems.InvoiceItemVariations',
+                'InvoiceTypes',
```

Comment:
I agree, it doesnt make sense to include a belongsTo-Association in the config. I made the change in InvoicesTable to demonstrate the SQL-error that you get when you do include it nevertheless (maybe you don't know better, like me when I first discovered this plugin). My goal was to provide a testcase for the change in DuplicateBehaviour.